athletics.lic

```diff
@@ -0,0 +1,170 @@
+custom_require.call(%w(common common-travel))
+
+class Athletics
+  
+  include DRC
+  include DRCT
+  
+  def initialize
+    @settings = get_settings()
+    @athletics_options = get_data('athletics').athletics_options
+
+    @climbing_target = get_data('athletics').practice_options[@settings.climbing_target]
+    
+    @start_skill=DRSkill.getxp('Athletics')
+    @end_skill=@start_skill+15
+    if(@end_skill>29)
+      @end_skill=29
+    end
+
+    if @climbing_target
+      walk_to(@climbing_target['id'])
+      climb_practice(@climbing_target['target'], @climbing_target['hide'])
+    elsif @settings.hometown == 'Crossing' || (@settings.hometown == 'Riverhaven' and DRSkill.getrank('Athletics') > 140)
+      crossing_athletics
+    elsif @settings.hometown == 'Shard'
+      shard_athletics
+    elsif @settings.hometown == 'Hibarnhvidar'
+      hib_athletics
+    elsif @settings.hometown == 'Ratha'
+      ratha_athletics
+    end
+  end
+  
+  def done_training?
+    return DRSkill.getxp('Athletics') >= @end_skill
+  end
+  
+  def crossing_athletics
+    if DRSkill.getrank('Athletics') >= 400
+      climb_branch
+    else
+      until done_training?
+        @athletics_options
+          .reject { |data| @settings.avoid_athletics_in_justice && data['justice'] }
+          .each do |data|
+            break unless climb?(data['room'], data['targets'])
+        end
+      end
+    end
+  end
+  
+  def shard_athletics
+    until done_training?
+      if DRSkill.getrank('Athletics') >120 and DRSkill.getrank('Athletics') <= 250
+        swim_zaldeni   
+      elsif DRSkill.getrank('Athletics') < 550
+        if DRSkill.getrank('Stealth') >= 250
+          #Shard side of undergondola.  Need to hide to practice there
+          climb_practice(19460,"wall",true)
+        else
+          swim_liirewsag
+        end
+      else
+        climb_branch
+      end
+    end
+  end
+  
+  def hib_athletics
+    swim_liirewsag
+  end
+  
+  def ratha_athletics
+    until done_training?
+      if DRSkill.getrank('Athletics') <= 185
+        climb_practice(5287, 'rock gorge',true)
+      else
+        if DRSkill.getrank('Stealth') >= 130
+          climb_practice(5143, 'deep crack',true)
+        end
+      end
+    end
+  end
+  
+  def swim_zaldeni
+    until done_training?
+      walk_to(11481)
+      walk_to(11482)
+      walk_to(11483)
+    end 
+  end
+  
+  def swim_liirewsag
+    #Intermittent errors in the map will cause go2 to get stuck.  The manual moves are a workaround for that
+    walk_to(4155)
+    move('nw')
+    until done_training?
+      fput('climb bank')
```

Comment:
Yes. For example: https://github.com/rpherbig/dr-scripts/blob/master/bescort.lic#L550